### ISSUE TITLE
Failed with `yarn start`

### ISSUE THREAD

**Issue by clearloop** [citahub/citahub-docs#91]
# Description

I was trying to fix a rendering bug last night, cloned this repo and started on my mac:
> _macOS Mojave, Version 10.14.3_

+ Default page `en-US/welcome` cannot GET.
+ The `community page` and `versions page` are okay, __without sidebar.__

I checked out the files of `citahub-docs`, in `citahub-docs/websites/pages` are just:
+ `en/community.js`
+ `en/versions.js`

__How can I fix this problem?__

# Commands
```
git clone https://github.com/cryptape/citahub-docs.git

cd citahub-docs/website

yarn && yarn run write-translations
yarn start
```

# Browser Display

### `/en-US/welcome`
![Screen Shot 2019-03-17 at 21 44 03 (2)](https://user-images.githubusercontent.com/26088946/54492021-f1132400-48fd-11e9-8183-8fe4c4f0411a.png)

### `/versions`
![Screen Shot 2019-03-17 at 21 53 49 (2)](https://user-images.githubusercontent.com/26088946/54492184-308e4000-48ff-11e9-8885-92b09a2d3283.png)

**Comment by kaikai1024** [comment]
@Keith-CY Help to take a look?

**Comment by zhouyun-zoe** [comment]
Thanks for your attention！We just fix it in #99 . You can have a try now 😁. Welcome more contributing!

**Comment by clearloop** [comment]
It works~ I'm closing this issue.
